### ISSUE TITLE
Unable to save a chart in PNG with python 2.7

### ISSUE THREAD

**Issue by MeTaNoV** [altair-viz/altair#994]
Hi,
congratulations for this very nice library.

I was trying to save a chart as PNG, but I am currently encountering the following error:

```
/home/pascal/Wks/GitHub/PEAT-AI/Albatros/venv/local/lib/python2.7/site-packages/altair/vegalite/v2/api.pyc in save(self, fp, format, override_data_transformer, scale_factor, **kwargs)
    412         if override_data_transformer:
    413             with data_transformers.enable('default', max_rows=None):
--> 414                 result = save(**kwds)
    415         else:
    416             result = save(**kwds)

/home/pascal/Wks/GitHub/PEAT-AI/Albatros/venv/local/lib/python2.7/site-packages/altair/utils/save.pyc in save(chart, fp, vega_version, vegaembed_version, format, mode, vegalite_version, embed_options, json_kwds, webdriver, scale_factor)
     92                                         vegaembed_version=vegaembed_version,
     93                                         webdriver=webdriver,
---> 94                                         scale_factor=scale_factor)
     95         if format == 'png':
     96             write_file_or_filename(fp, mimebundle['image/png'], mode='wb')

/home/pascal/Wks/GitHub/PEAT-AI/Albatros/venv/local/lib/python2.7/site-packages/altair/utils/mimebundle.pyc in spec_to_mimebundle(spec, format, mode, vega_version, vegaembed_version, vegalite_version, **kwargs)
     56                               vegalite_version=vegalite_version, **kwargs)
     57         if format == 'png':
---> 58             render = base64.decodebytes(render.split(',', 1)[1].encode())
     59             return {'image/png': render}
     60         elif format == 'svg':

AttributeError: 'module' object has no attribute 'decodebytes'
```
It looks like `decodebytes` method is only available starting from Python 3.1
https://docs.python.org/3/library/base64.html
vs
https://docs.python.org/2/library/base64.html

Cheers,
Pascal

**Comment by jakevdp** [comment]
Thanks, looks like we missed that. Strangely it doesn't look like six has a backport for ``decodebytes``...

**Comment by MeTaNoV** [comment]
@jakevdp , a PR is on its way, but it doesn't look like the repository is actively maintained by its owner...

**Comment by pratapvardhan** [comment]
I suspect `(base64.decodebytes if six.P3 else base64.b64decode)(render.split(',', 1)[1].encode())` might be able to work. I hope there is clear way around.

**Comment by jakevdp** [comment]
Either of those solutions looks great.

The reason we didn't catch this is because we haven't yet set up travis to support headless rendering, so it only gets tested locally (and most of our developers are running Python 3 I think).

**Comment by jimmywan** [comment]
I think this can be closed since #1134 got merged.
